src/core/Form/Toggle/Toggle.md

```diff
@@ -26,5 +26,12 @@ import { Toggle } from 'suomifi-ui-components';
   <Toggle onClick={({ toggleState }) => console.log(toggleState)}>
     Unchecked enabled using button
   </Toggle>
+
+  <Toggle
+    checked={false}
+    onClick={({ toggleState }) => console.log(toggleState)}
+  >
+    Controlled checked state.
+  </Toggle>
```

Comment:
Maybe add use case of controlled version, to see it in action. Something like this might be nice:

```jsx
import { useState } from 'react';
//...
const [toggleChecked, setToggleChecked] = useState(false);
//...
  <Toggle
    checked={toggleChecked}
    onClick={({ toggleState }) => console.log(toggleState)}
  >
    Controlled checked state.
  </Toggle>
  <button onClick={() => setToggleChecked(!toggleChecked)}>
    Toggle the controlled version
  </button>
```

Reply:
Tried this but could not get it to work, perhaps due to a bug that is now fixed. Will add something like this to better describe how the component works.